release-notes.md

```diff
@@ -1,5 +1,16 @@
 # Release notes
 
+## July 31, 2019
+
+**Places Monitor 2.0.0**
+
+* Monitoring status is now persisted between launches.
+* Handling of the callback resulting from a location permission request no longer requires you to extend PlacesActivity.
+* Changed an existing API, allowing developers to clear all Places data from the device
```

Comment:
- please put a colon after "device" and put the Old AP and New API information as 2nd-level bullets